### PR TITLE
Remove unwrap

### PR DESCRIPTION
**Pull request by cryptonemo**
This is an updated version of #1122, as a lot of code has changed since then

### CODE REVIEW

File: fil-proofs-tooling/src/bin/benchy/main.rs [hunk 1]
```diff
@@ -1,3 +1,6 @@
+//requires nightly, or later stable version
+//#![warn(clippy::unwrap_used)]
```
Comment (vmx) [inline fil-proofs-tooling/src/bin/benchy/main.rs:2]: This might be possible now. I haven't checked though.

Comment (vmx) [reply fil-proofs-tooling/src/bin/benchy/main.rs:2]: That surely isn't a blocker to get this merged.

Comment (cryptonemo) [reply fil-proofs-tooling/src/bin/benchy/main.rs:2]: I think it works fine on later stable, definitely not on the toolchain we're using though.  If/when we upgrade, we can add those back in.  I think that's a separate discussion from this PR though.

**Comment by vmx** [comment]
I think we should merge this one soon, before it gets outdated again.

**Comment by cryptonemo** [comment]
@vmx Agreed. Also, related to #1261 @porcuquine Please review.

**Pull request merged**